rpki-validator/src/main/java/net/ripe/rpki/validator3/domain/validation/ValidatedRpkiObjects.java

```diff
@@ -299,6 +300,6 @@ private long countRoaPrefixes(SearchTerm searchTerm) {
 
     private void notifyListeners() {
         Locks.locked(dataLock.readLock(), () ->
-                listeners.forEach(listener -> listener.accept(validatedObjectsByTrustAnchor.values())));
+            listeners.forEach(listener -> listener.accept(validatedObjectsByTrustAnchor.values())));
```

Comment:
Not a change introduced here - but I noticed that `listener.accept` is sometimes called under writelock (see `.addListener`) and sometimes under readlock.

Reply:
Oh, that's a very good catch!
notifyListeners should be in the readLock of course, listeners read the data.

Reply:
Fixed.